src/Options/OptionsDetails.hpp

```diff
@@ -68,7 +71,9 @@ struct is_in_group<Tag, Group, std::void_t<typename Tag::group>>
 
 /// The subset of tags in `OptionList` that are in the hierarchy of `Group`
 template <typename OptionList, typename Group>
-using options_in_group = tmpl::filter<OptionList, is_in_group<tmpl::_1, Group>>;
+using options_in_group =
+    tmpl::filter<OptionList,
+                 tmpl::bind<is_in_group, tmpl::_1, tmpl::pin<Group>>>;
```

Comment:
can't you just `pin` the `Group` above? `bind` just turns the type alias back into a struct, essentially.